documentation/smg/16-serialiser.md

```diff
@@ -0,0 +1,275 @@
+# Serialisation details
+
+As of the integration of the C++ serialiser, the serialised format has been updated to be more consistent and to have greater capabilities.
```

Comment:
Merging the serialiser into Herbert.

Reply:
It would be good to say that in the text. In general, if it is not obvious to the reviewer what you are referring to, then it is worth adding to the text to make it clear. In this case, you could say "In the context of merging the serialiser into Herbert, the serialised format has been updated ...." This also begs the question of where the serialiser was before so that a merge into Herbert is needed, and you could also briefly add that information,